py-rattler/src/component.rs

```diff
@@ -0,0 +1,30 @@
+use pyo3::{IntoPy, PyObject, Python};
```

Comment:
Maybe it would be nice to turn the `version.rs` file into a module and add this file there as well. Especially since `PyComponent` is a rather generic name it might be nice to add it to a module, to give it some more context.